components/shared/TokenPicker.tsx

```diff
@@ -90,66 +83,121 @@ const TokenPicker: React.FC<{
 				) {
-					selectedToken === "ETH"
+					tokens[selectedTokenIdx]?.symbol === "ETH"
 						? setToken({ address: ETH, symbol: "ETH", decimals: 18 })
 						: setToken(token);
 				}
 			});
 		}
-	}, [cennznet, assets, selectedToken, setToken]);
+	}, [cennznet, assets, selectedTokenIdx, tokens]);
+
+	useEffect(() => {
+		if (!balances || !tokens) return;
+		const foundTokenBalance = balances.find(
+			(asset) => asset.symbol === tokens[selectedTokenIdx]?.symbol
+		);
+		setSelectedTokenBalance(foundTokenBalance.value);
+	}, [balances, tokens]);
 
 	return (
-		<FormControl
-			sx={{
-				width: "80%",
-				mt: "30px",
-			}}
-			required
-			disabled={router.asPath === "/bridge" ? (Account ? false : true) : false}
-		>
-			<InputLabel>Token</InputLabel>
-			<Select
-				required
-				value={selectedToken}
-				onChange={(e) => {
-					setSelectedToken(e.target.value);
+		<div className={styles.tokenPickerContainer}>
+			<FormControl
+				sx={{
+					width: "142px",
 				}}
-				input={<OutlinedInput label="Token" />}
-				MenuProps={MenuProps}
-				sx={{ fontSize: "18px" }}
+				disabled={router.asPath === "/bridge" ? !Account : false}
 			>
-				{tokens?.map((token: any, i) => (
-					<MenuItem
-						key={i}
-						value={token.symbol}
-						sx={{
-							fontSize: "18px",
-						}}
-					>
-						<img
-							key={`img ${token.logo}`}
-							alt="token logo"
-							src={token.logo}
-							style={{ marginRight: "12px", width: "20px" }}
-						/>
-						{token.symbol}
-					</MenuItem>
-				))}
-			</Select>
-		</FormControl>
+				<div className={styles.tokenSelector}>
+					{assetsLoading ? (
+						<CircularProgress />
+					) : (
+						<>
+							<img
+								className={styles.tokenSelectedImg}
+								alt=""
+								src={tokens[selectedTokenIdx]?.logo}
+								width={33}
+								height={33}
+							/>
+							<button
+								type="button"
+								className={styles.tokenButton}
+								onClick={() => setTokenDropDownActive(!tokenDropDownActive)}
+							>
+								{tokens[selectedTokenIdx]?.symbol}
+								<img
+									className={
+										tokenDropDownActive
+											? styles.tokenSelectedArrow
+											: styles.tokenSelectedArrowDown
+									}
+									alt="arrow"
+									src={"/arrow_up.svg"}
+								/>
+							</button>
+						</>
+					)}
+					{tokenDropDownActive && (
+						<div className={styles.tokenDropdownContainer}>
+							{tokens.map((token: any, i) => {
+								return (
+									<div
+										key={i}
+										onClick={() => {
+											setSelectedTokenIdx(i);
+											setTokenDropDownActive(false);
+										}}
+										className={styles.tokenChoiceContainer}
+									>
+										<img alt="" src={token.logo} width={33} height={33} />
+										<span>{token.symbol}</span>
+									</div>
+								);
+							})}
+						</div>
+					)}
+				</div>
+			</FormControl>
+			<div className={styles.amountContainer}>
+				<Button
+					sx={{
+						fontFamily: "Roboto",
```

Comment:
I've seen this across different components, do we need to specify `fontFamily` explicitly for some components? @eupston @aidan-starke

Reply:
not as far as I know, we could also remove the fonts file from public, and the references to Teko in theme/theme.ts and theme/StyledComponents.ts